base_location_geonames_import/wizard/geonames_import.py

```diff
@@ -75,26 +67,19 @@ def _prepare_res_city(self, row, country):
         return vals
 
     @api.model
-    def _prepare_better_zip(self, row, country, res_city):
-        state = self.select_or_create_state(row, country)
-        city_name = self.transform_city_name(row[2], country)
+    def _prepare_city_zip(self, row, res_city):
         vals = {
             'name': row[1],
             'city_id': res_city and res_city.id or False,
-            'city': res_city and res_city.name or city_name,
-            'state_id': state.id,
-            'country_id': country.id,
-            'latitude': row[9],
-            'longitude': row[10],
         }
         return vals
 
     @api.model
-    def create_better_zip(self, row, country, res_city):
+    def create_city_zip(self, row, country, res_city):
         if row[0] != country.code:
             raise UserError(
                 _("The country code inside the file (%s) doesn't "
-                    "correspond to the selected country (%s).")
+                  "correspond to the selected country (%s).")
                 % (row[0], country.code))
         logger.debug('ZIP = %s - City = %s' % (row[1], row[2]))
         if self.letter_case == 'title':
```

Comment:
You should put this include this in `transform_city_name` method and call it everywhere for consistency